governance/program/tests/program_test/addins.rs

```diff
@@ -0,0 +1,84 @@
+// Governance program has no dependency on the voter-weight-addin program and hence we can't use its instructions here
+// The instructions have to be manually recreated
```

Comment:
You could save yourself some rewriting by defining the instruction interface and creator in the governance program, and having the sample addin program just process that instruction.

Reply:
yes, that would be possible, but since the `addin` is not a real one I didn't want to pollute `governance` program with this mock interface.
I also renamed the `addin` program to `addin-mock` to make the intention explicit